### ISSUE TITLE
Unable to initialise multitenancy to the Blazor App 

### ISSUE THREAD

**Issue by CroutledgeDatasym** [OrchardCMS/OrchardCore#16829]
### Orchard Core version

Contentmanagement - version 2.0.2

Following the guide - https://docs.orchardcore.net/en/latest/guides/create-blazor-cms/#__tabbed_4_2

### To Reproduce
Steps to reproduce the behaviour:
I followed the guide - "Create a Decoupled CMS site with a .NET Blazor Application (Server Side)

I get to the stage "Add multitenancy to the Blazor App" and create the FirstTenant example, which is created but when i try to set it up it takes me straight to "https://localhost:XXXX/tenant01?token=....." and skips the "Setup" page therefore none of the database table are created

### Expected behaviour
I would expect it to follow the guide and take me to the Setup page (unless I am misunderstanding how Multitenancy works)

### Logs and screenshots
Not too sure what logs would be applicable but can provide if needed

**Comment by hishamco** [comment]
@psijkof works on something similar if I remember

**Comment by psijkof** [comment]
Yes, thank you for reminding me to highlight an important issue.

Orchard Core always listens to / (also a tenant root, e.g. /firsttenant/).

Therefore, to avoid conflicts and 'normal' orchard responses, change your home.razor page, to use 

`@page "/home"` for example, and try again the sample. (Note, that you probably also want to change your nav menu to use /home instead of /)

HTH

**Comment by CroutledgeDatasym** [comment]
@psijkof  Thank you, that has helped resolved the issue I was having . Very much appreciate the quick response!
